### PR TITLE
zh-CN: translate generics

### PR DESCRIPTION
**Pull request by mgeisler**
Part of #324.

### CODE REVIEW

**Comment by mgeisler** [comment]
Hey all, please take a look and leave suggestions if there is something here I should fix.

File: po/zh-CN.po [hunk 1]
```diff
@@ -9014,11 +9015,11 @@ msgstr ""
 
 #: src/generics/methods.md:1
 msgid "# Generic Methods"
-msgstr ""
+msgstr "# 通用方法"
```
Comment (wnghl) [inline po/zh-CN.po:9018]:
```suggestion
msgstr "# 泛型方法"
```
Maybe '泛型' is a more common translation of word 'generic'

File: po/zh-CN.po [hunk 2]
```diff
@@ -9052,14 +9053,20 @@ msgid ""
...
 
 #: src/generics/monomorphization.md:1
 msgid "# Monomorphization"
-msgstr ""
+msgstr "# 单态化"
 
 #: src/generics/monomorphization.md:3
 msgid "Generic code is turned into non-generic code based on the call sites:"
-msgstr ""
+msgstr "通用代码会根据调用站点转换为非通用代码："
```
Comment (wnghl) [inline po/zh-CN.po:9069]:
```suggestion
msgstr "泛型代码根据调用位置转换为非泛型代码："
```

File: po/zh-CN.po (hunk 1 above)
Comment (mgeisler) [reply po/zh-CN.po:9018]: Let me commit your suggestion for now, then I suggest discussion the best translation on #324 or similar.

**Review by wnghl — APPROVED** [review]

**Pull request merged**